.github/actions/test/action.yaml

```diff
@@ -7,29 +7,38 @@ runs:
     run: cargo test --manifest-path tools/slicec-cs/Cargo.toml
     shell: bash
 
-  - name: 🧪 Test
+  - name: 🧪 Test .NET
     run: |
       mkdir -p TestResults
-      dotnet test --no-build --verbosity normal --blame-hang-timeout 10s --logger "trx" --results-directory TestResults
+      dotnet test \
+        --no-build \
+        --verbosity normal \
+        --blame \
+        --blame-crash \
+        --blame-hang-timeout 300s \
```

Comment:
300 s (5 minutes) sound quite long. My understanding it's for a single test.

Maybe 10 seconds was too short, and 30 or 60 seconds makes more sense?